# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1465237337 25200
#      Mon Jun 06 11:22:17 2016 -0700
# Node ID cbd8cf0a8a5c3fab7d65548a50c61c124d7c2836
# Parent  d9fa4558d2ed4a21a94aa1181a6dd586bf1252bd
Fix psi input validation with --enable-64

* psi.cc: Fix error message validation regexp for --enable-64 build.

diff --git a/libinterp/corefcn/psi.cc b/libinterp/corefcn/psi.cc
--- a/libinterp/corefcn/psi.cc
+++ b/libinterp/corefcn/psi.cc
@@ -217,15 +217,15 @@ can have any value real or complex value
 %!   assert (psi (n, z+1),
 %!           psi (n, z) + ((-1)^n) * factorial (n) * (z.^(-n-1)), 0.1);
 %! endfor
 
 ## Test input validation
 %!error psi ()
 %!error psi (1, 2, 3)
 %!error <Z must be> psi ("non numeric")
-%!error <conversion of 5.3 to int value failed> psi (5.3, 1)
+%!error <conversion of 5.3 to int.* value failed> psi (5.3, 1)
 %!error <K must be non-negative> psi (-5, 1)
 %!error <Z must be non-negative for polygamma> psi (5, -1)
 %!error <Z must be a floating point> psi (5, uint8 (-1))
 %!error <Z must be real value for polygamma> psi (5, 5i)
 
 */
